docs/classes.rst

```diff
@@ -37,8 +37,14 @@ The binding code for ``Pet`` looks as follows:
 :class:`class_` creates bindings for a C++ *class* or *struct*-style data
 structure. :func:`init` is a convenience function that takes the types of a
 constructor's parameters as template arguments and wraps the corresponding
-constructor (see the :ref:`custom_constructors` section for details). An
-interactive Python session demonstrating this example is shown below:
+constructor (see the :ref:`custom_constructors` section for details).
+
+.. note::
+
+    Starting with pybind11v3, it is recommended to use `py::classh` in most
```

Comment:
Maybe it's another PR, but since it's the recommended default, we probably should change *all* of the documentation to use it, since users often just copy/paste from the documentation without thinking about it.

Reply:
I was on the fence, thanks for the feedback! I'll change it around.

Reply:
Done in commit ac9d31e13fd15f2a8e3deeccd840e5038c028da8.

(I made a fast pass, followed by a careful slow pass reviewing the generated html.)